shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -116,45 +112,35 @@ public int cat(TachyonURI path) throws IOException {
    * Load a file or directory in Tachyon space, makes it resident in memory.
```

Comment:
Load -> Loads.
Since you're on this, could you also fix the docs of the method and others in this class?